app/CocoaPods/CPPodfileInitController.swift

```diff
@@ -23,8 +23,9 @@ public class CPPodfileInitController: NSObject, CPCLITaskDelegate {
     self.projectURL = xcodeprojURL
     
     super.init()
+
     let task = CPCLITask(workingDirectory: xcodeprojURL.URLByDeletingLastPathComponent!.path,
-      command: "init",
+      command: "init \(xcodeprojURL.lastPathComponent!)",
```

Comment:
You can't select an empty file/folder from the UI, so I think this is safe ( I saw it above and went through the same thinking process )